WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeButtonPrefsFragment.java

```diff
@@ -73,8 +73,6 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ((WordPress) getActivity().getApplication()).component().inject(this);
 
-        setRetainInstance(true);
```

Comment:
I see your point! The issue is that we can't rely on `setRetainInstance(true)` when activity is destroyed as global variables like `mSite ` are also destroyed (it only works as intended on configuration change). We have to retain `mSite` no matter what, so that's why I removed the `setRetainInstance(true)`. 
Maybe @nbradbury (based on `PublicizeButtonPrefsFragment` edit history) can take a look and confirm that everything works as intended.

Reply:
Fair point. So, yeah, let's get @nbradbury 's opinion about removing `setRetainInstance(true)` and we'll take it from there.

@nbradbury , what do you think? Would removing the fragment retainment here have any side effects we don't see? Thanks!

Reply:
I'm trying to recall why I added `setRetainInstance(true)` here, because like @hypest I'm not a fan of it. I'm annoyed I added that without a comment explaining why it was necessary, but I know if I added it then it *was* necessary so I don't think it should be removed.